### ISSUE TITLE
[Bug] Shorts are slipping through even tho I disabled them

### ISSUE THREAD

**Issue by Harmonie666** [YTVanced/Vanced#1214]
**Bug description**
Replace me with a clear and detailed description of the bug.

**Variant**
Non-Root

**Vanced version**
Version 16.29.39

**Android version**
Android 10 

**Device**
ROG Phone 2

![Screenshot_20210823-013555568 (1).png](https://user-images.githubusercontent.com/73917917/130373752-432b69cd-44d4-487e-bc9f-b97a8a6a5553.png)![Screenshot_20210823-013644474.png](https://user-images.githubusercontent.com/73917917/130373753-afc7c7e6-7a72-406a-87cd-a3e3be0f4f70.png)![Screenshot_20210823-013656956 (1).png](https://user-images.githubusercontent.com/73917917/130373756-d556439d-dd09-4fb2-941c-a63fa2ef1683.png)

**Comment by milindgoel15** [comment]
No, these are a new form of shorts shelf. The existing removal method works on the vertical type of shorts shelf only.

**Comment by Harmonie666** [comment]
> No, these are a new form of shorts shelf. The existing removal method works on the vertical type of shorts shelf only. And only few of the users have got this type of shorts.

Got it. Are you able to remove these "new form" of shorts aswell? That would be great :)

**Comment by milindgoel15** [comment]
Dunno if xfile got it or not. So wait for future updates

**Comment by Harmonie666** [comment]
Also reddit post with tons of upvotes:

https://www.reddit.com/r/Vanced/comments/p9noa7/shorts_cringe_are_slipping_through_even_tho_i/?utm_medium=android_app&utm_source=share

**Comment by milindgoel15** [comment]
Upvotes are useless if dev can't get it himself in the first place.

**Comment by Harmonie666** [comment]
Of course :)
Hopefully Dev will be able to remove them aswell.
